src/nav-docs.yml

```diff
@@ -77,4 +77,8 @@ children:
         path: '/working-with-secrets'
       - title: 'Virtual modules'
         path: '/virtual-modules'
+      - title: 'SVG React components'
```

Comment:
Will these get pulled into internal documentation?

Reply:
Yeah, this happens automatically